internal/store/postgres/resource_repository.go

```diff
@@ -74,7 +74,8 @@ func applyResourceFilter(db *gorm.DB, filter domain.ListResourcesFilter) *gorm.D
 		// Otherwise, it will be wrapped in parentheses and the query will be invalid.
 		db = db.Where(db.
 			Where(`LOWER("urn") LIKE ?`, fmt.Sprintf("%%%s%%", strings.ToLower(filter.Q))).
-			Or(`LOWER("name") LIKE ?`, fmt.Sprintf("%%%s%%", strings.ToLower(filter.Q))),
+			Or(`LOWER("name") LIKE ?`, fmt.Sprintf("%%%s%%", strings.ToLower(filter.Q))).
```

Comment:
you can use `ILIKE` to be case insensitive instead of transforming to lowercase which causes dependency on the field to always be stored in lowercase.

https://www.geeksforgeeks.org/postgresql-ilike-operator/

```suggestion
			Or(`"name" ILIKE ?`, fmt.Sprintf("%%%s%%", strings.ToLower(filter.Q))).
```